src/app/devices/deviceContent/cloudToDeviceMessage/saga.ts

```diff
@@ -65,3 +63,7 @@ export function* cloudToDeviceMessageSaga(action: Action<CloudToDeviceMessageAct
         yield put(cloudToDeviceMessageAction.failed({params: action.payload, error}));
     }
 }
+
+export function* cloudToDeviceMessageSagaWatcher() {
```

Comment:
I am curious about the rule that we shall follow in all sagas, worker/watch/normal saga. Compared with saga.ts in addDevice

Reply:
agreed. Please set a rule. will follow